[{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python3-os-faults",
	"source": "python-os-faults",
	"version": "0.2.1-3",
	"section": "python",
	"priority": "optional",
	"architecture": "all",
	"depends": [
		"python3-appdirs (>= 1.3.0)",
		"python3-click",
		"python3-iso8601",
		"python3-jsonschema",
		"python3-oslo.concurrency",
		"python3-oslo.i18n",
		"python3-oslo.serialization",
		"python3-oslo.utils",
		"python3-pbr",
		"python3-pyghmi",
		"python3-six (>= 1.9.0)",
		"python3-yaml",
		"python3:any"
	],
	"suggests": [
		"python-os-faults-doc"
	],
	"installedSize": 305,
	"maintainer": "Debian OpenStack <team+openstack@tracker.debian.org>",
	"description": "OpenStack fault-injection library - Python 3.x",
	"homepage": "http://os-faults.readthedocs.io/",
	"filename": "pool/main/p/python-os-faults/python3-os-faults_0.2.1-3_all.deb",
	"size": 37156,
	"md5": "de0c8833f1a4087944dcce68b6f1f1bc",
	"sha256": "071607f2d865322ea6572ce64d3fa515388d9c3e4aac20c0201ad6f2a3f56656",
	"descriptionMd5": "bebdf221db0eec0dc9b875357743a265"
}]